src/channel.ts

```diff
@@ -1476,6 +1482,11 @@ export class Channel<
         }
         break;
       case 'message.new':
+        s.unreadCount = s.unreadCount + 1;
```

Comment:
```suggestion
        s.unreadCount = event.unread_count;
```
the same thing might need to be applied to `message.deleted`.

Reply:
my guess is we return this value in some other channel related events too, we can check for the existence of this value at the beginning of the switch-case and keep it updated with all the events

Reply:
`unread_count ` on event object is the total unread count (across all channels)!!

Reply:
@mahboubii ^^